modules/ln_glgui/glgui.scm

```diff
@@ -250,9 +250,23 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ;;
 ;; Examples:
 ;;
-;;  (glgui-timings-set! frame-period-max: 2 frame-period-min: 0.01) -- start with 110/sec down to 0.5/sec
+;;  (glgui-timings-set! frame-period-max: 2 frame-period-min: 0.01) -- start with 100/sec down to 0.5/sec
 ;;  (glgui-timings-set! frame-period-max: 0.5 frame-period-min: 0.05) -- 20/sec down to 2/sec
 ;;  (glgui-timings-set! frame-period-custom: (lambda (x) 0.01))  -- constant 100/sec
+;;
+;;  When concerned about a precise frame rate take a hint from gambit
+;;  manual wrt. `thread-sleep!` and wait until a point in time no
+;;  matter how long it took to process events and render the GUI:
+;;
+;; (glgui-timings-set!
```

Comment:
I think we should offer a function for people to call that does provide a reset to the old (well slightly improved) timing behaviour using this piece of code. That way we don't end up with slightly different versions of it everywhere and it can be updated in one place. So a `glgui-timings-at-10msec!` with a better name?

Reply:
How about `(glgui-timings-set!)` with no keyword set at all?  Trivial to implement (just one `(or frame-period-max frame-period-min frame-period-custom)` to select the branch modifying the setting and resetting them in case non is given?

No second global binding, nothing.  Just return to defaults.  And only required if anything did change anything before.

Reply:
No, I want to have a function that when I call it we get the precise 10msec timing (well the improved version) back, not revert all settings to the new defaults.

Reply:
Like this:

```
(define (glgui-timings-at-10msec!)
  (glgui-timings-set! frame-period-custom: (lambda (_) 0.01))
```
That should do the trick.

This should do it as well:

```
(define (glgui-timings-at-10msec!)
  (glgui-timings-set! frame-period-custom: #f frame-period-max: 0.01  frame-period-min: 0.01)
```
Which name should it be?

- glgui-timings-at-10msec!
- glgui-timings-traidtional!
- ...

Reply:
I was referring to the new piece you made with the `(next (+ last-frame-time frame-period)` in it. As for naming, 10msec is more clear what it does, but calling it traditional or legacy also would work - no strong preference here.